src/vue/pages/sale-details/SaleCampaign.vue

```diff
@@ -0,0 +1,324 @@
+<template>
+  <div class="sale-campaign">
+    <template v-if="isLoaded">
+      <drawer :is-shown.sync="isInvestDrawerShown">
+        <template slot="heading">
+          {{ 'sale-details.invest' | globalize }}
+        </template>
+
+        <invest-form
```

Comment:
you're duplicating the same drawer here and in the top bar. You can either leave only top one and emit an event to open it or remove the button from here at all because it is obsolete in the design I think

Reply:
Removed invest button